### PR TITLE
Fixed overflow

### PR DESCRIPTION
**Pull request by JM066**
# Description

Overflow on the user page is fixed

## Type of change

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings

## Showcase

Screenshot from 2022-03-01 23-25-35

### CODE REVIEW

**Comment by ivanms1** [comment]
Task linked: [CU-2az11a5 fix overflow in user page](https://app.clickup.com/t/2az11a5)

File: apps/web/src/pages/User/styles.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const StyledUser = styled('div', {
   flexDirection: 'column',
   backgroundColor: '#F9F8FD',
   padding: 26,
-  height: '100vh',
+  // height: '100vh',
```
Comment (ivanms1) [inline apps/web/src/pages/User/styles.ts:10]: remove please

File: apps/web/src/pages/User/styles.ts [hunk 2]
```diff
@@ -21,11 +21,10 @@ export const StyledUserContainer = styled('div', {
...
+  width: '100%',
 });
 
 export const StyledProjectContainer = styled('div', {
-  display: 'flex',
-  flexDirection: 'row',
-  gap: 20,
-  padding: 26,
+  width: '100%',
+  padding: '40px 100px',
```
Comment (ivanms1) [inline apps/web/src/pages/User/styles.ts:29]: is this mobile friendly?

**Comment by ivanms1** [comment]
please read the last part of the readme template :)

File: apps/web/src/pages/User/styles.ts (hunk 1 above)
Comment (JM066) [reply apps/web/src/pages/User/styles.ts:10]: ok

File: apps/web/src/pages/User/styles.ts (hunk 2 above)
Comment (JM066) [reply apps/web/src/pages/User/styles.ts:29]: oh no.. I don't know what happened there

**Pull request merged**